### DISCUSSION TITLE
Metrics for a specific repository don't appear to be accurate

### DISCUSSION THREAD

**Discussion by binford2k** [community/OSPO#23] [Dashboard]
Our dashboard is at https://github.com/orgs/puppetlabs/insights/org_metrics

One specific [repository](https://github.com/puppetlabs/puppetlabs-cd4pe_tests) in there shows up as
- 99.9% external contributions
- most recent activity of two days ago
- 3247 contributions

But those don't match the stats of the repository itself! https://github.com/puppetlabs/puppetlabs-cd4pe_tests/graphs/contributors

Where do the stats come from?

**Comment by mrbobbytables** [comment]
We have the same issue with k8s, for us it lists things as 'last contribution 19 days ago', but we've had PRs merged even as of a few seconds ago.

**Comment by ahpook** [comment]
to @binford2k's original point - the way we were computing "external contributions" was pretty flawed - see [this discussion](https://github.com/community/OSPO/discussions/35) for more info; we've since removed that entirely.

The "most recent activity" is based on a periodic query and won't be up-to-the-minute ; I'll ask one of the devs to keep me honest on this but I think it'll always be at least two days - this is probably worth a docs caveat.  

That said though, @mrbobbytables 19 days seems excessive and worth some more investigation on our end. Is it still showing that? And does it seem like it's not refreshing at all (as in, now it's 23 days)? Or is it a constant "19 days ago" as the newest activity?

**Comment by mrbobbytables** [comment]
@ahpook it's back to saying 2 days for most. No idea why it was saying that before O_o

**Reply by ahpook** [reply]
That's an improvement anyway 😬

**Comment by mrbobbytables** [comment]
FWIW - its back to saying 9 days. Time to update seems very inconsistent.

**Reply by ahpook** [reply]
Is this fixed for you? I was seeing the same in other orgs until yesterday evening and now it's back to normal - these stats were affected by some of the availability pains GitHub has been experiencing the past couple of weeks :(  (Which fortunately seem to be on the mend)

**Reply by mrbobbytables** [reply]
Yup - now back to 2 days. 👍
